diff -Nru vdr-1.3.11/svdrp.c vdr-1.3.11.patch/svdrp.c
--- vdr-1.3.11/svdrp.c	Sun Jun 13 15:38:38 2004
+++ vdr-1.3.11.patch/svdrp.c	Thu Aug 26 14:41:54 2004
@@ -256,6 +256,8 @@
   "    format defined in vdr(5) for the 'epg.data' file.  A '.' on a line\n"
   "    by itself terminates the input and starts processing of the data (all\n"
   "    entered data is buffered until the terminating '.' is seen).",
+  "RENR <number> <new name>\n"
+  "    Rename recording. Number must be the Number as returned by LSTR command.",
   "SCAN\n"
   "    Forces an EPG scan. If this is a single DVB device system, the scan\n"
   "    will be done on the primary device unless it is currently recording.",
@@ -1025,7 +1027,33 @@
   EITScanner.ForceScan();
   Reply(250, "EPG scan triggered");
 }
+void cSVDRP::CmdRENR(const char *Option)
+{
+  bool recordings = Recordings.Load();
+  if (recordings) {
+    if (*Option) {
+       char *tail;
+       int n = strtol(Option, &tail, 10);
+       cRecording *recording = Recordings.Get(n - 1);
+       if (recording && tail && tail != Option) {
+         tail = skipspace(tail);
+         int priority=recording->priority;
+         int lifetime=recording->lifetime;
+         recording->Rename(tail,&priority,&lifetime);
+         Reply (250,"Renamed \"%s\" to \"%s\"",recording->Name(),tail);
+  //       Reply (200,"New Name: %s",tail);
+         }
+       else
+         Reply(501, "Recording not found or wrong syntax");
+       }
+    else
+       Reply(501, "Missing Input settings");
+  }
+  else
+     Reply(550, "No recordings available");
+}
 
+						
 void cSVDRP::CmdSTAT(const char *Option)
 {
   if (*Option) {
@@ -1133,6 +1161,7 @@
   else if (CMD("NEWT"))  CmdNEWT(s);
   else if (CMD("NEXT"))  CmdNEXT(s);
   else if (CMD("PUTE"))  CmdPUTE(s);
+  else if (CMD("RENR"))  CmdRENR(s);
   else if (CMD("SCAN"))  CmdSCAN(s);
   else if (CMD("STAT"))  CmdSTAT(s);
   else if (CMD("UPDT"))  CmdUPDT(s);
diff -Nru vdr-1.3.11/svdrp.h vdr-1.3.11.patch/svdrp.h
--- vdr-1.3.11/svdrp.h	Sat Jan 17 14:30:52 2004
+++ vdr-1.3.11.patch/svdrp.h	Thu Aug 26 14:41:54 2004
@@ -73,6 +73,7 @@
   void CmdNEWT(const char *Option);
   void CmdNEXT(const char *Option);
   void CmdPUTE(const char *Option);
+  void CmdRENR(const char *Option);
   void CmdSCAN(const char *Option);
   void CmdSTAT(const char *Option);
   void CmdUPDT(const char *Option);
