--- vdradmind.pl.save	2005-03-20 19:38:06.000000000 +0100
+++ vdradmind.pl	2005-03-20 19:43:51.000000000 +0100
@@ -1435,26 +1435,25 @@
         substr($stop, 0, 2), $stop > $start ? $3 : $3 + 1,
         ($2 - 1), $1);
     } else { # regular timer
-      $startsse = my_mktime(substr($start, 2, 2),
-				substr($start, 0, 2), $dor, (my_strftime("%m") - 1),
-				my_strftime("%Y"));
+      $dor =~ /(\d\d\d\d)-(\d\d)-(\d\d)/;
+      $startsse = my_mktime(substr($start, 2, 2), substr($start, 0, 2), $3, ($2 - 1), $1);
 			
       $stopsse = my_mktime(substr($stop, 2, 2),
-        substr($stop, 0, 2), $stop > $start ? $dor : $dor + 1,
-        (my_strftime("%m") - 1), my_strftime("%Y"));
+        substr($stop, 0, 2), $stop > $start ? $3 : $3 + 1, ($2 - 1), $1);
     }
 
-    # move timers which have expired one month into the future
-    if(length($dor) != 7 && $stopsse < time) {
-			$startsse = my_mktime(substr($start, 2, 2),
-        substr($start, 0, 2), $dor, (my_strftime("%m") % 12),
-        (my_strftime("%Y") + (my_strftime("%m") == 12 ? 1 : 0)));
-
-      $stopsse = my_mktime(substr($stop, 2, 2),
-        substr($stop, 0, 2), $stop > $start ? $dor : $dor + 1,
-        (my_strftime("%m") % 12),
-        (my_strftime("%Y") + (my_strftime("%m") == 12 ? 1 : 0)));
-    }
+#    vdr-1.3.23 changes day format to yyyy-mm-dd
+#    # move timers which have expired one month into the future
+#    if(length($dor) != 7 && $stopsse < time) {
+#			$startsse = my_mktime(substr($start, 2, 2),
+#        substr($start, 0, 2), $dor, (my_strftime("%m") % 12),
+#        (my_strftime("%Y") + (my_strftime("%m") == 12 ? 1 : 0)));
+#
+#      $stopsse = my_mktime(substr($stop, 2, 2),
+#        substr($stop, 0, 2), $stop > $start ? $dor : $dor + 1,
+#        (my_strftime("%m") % 12),
+#        (my_strftime("%Y") + (my_strftime("%m") == 12 ? 1 : 0)));
+#    }
 
     if($CONFIG{RECORDINGS} && length($dor) == 7)  { # repeating timer
       # generate repeating timer entries for up to 28 days
